etl2/tests/data_tests/test_join_counts_deputies.py

```diff
@@ -4,42 +4,43 @@
 
 def test_joins_deputies():
 
-    # casrec_db_connection = psycopg2.connect(
-    #     "host=localhost port=6666 "
-    #     "dbname=casrecmigration "
-    #     "user=casrec "
-    #     "password=casrec"
-    # )
-
-    etl1_query = (
-        "select "
-        '"Case" as caserecnumber, "Forename" as firstname, "Surname" as surname, '
-        "1 as address_count, "
-        '(select count(*) from etl1."order" where "Case" = etl1.pat."Case") as '
-        "cases "
-        "from etl1.deputy;"
+    casrec_db_connection = psycopg2.connect(
```

Comment:
This stuff need parameterising but i can do it as part of AWS-ifying it

Reply:
Arg the main stuff is done but not the tests.
Next PR :)